oapi_validate.go

```diff
@@ -19,14 +20,26 @@ import (
 // ErrorHandler is called when there is an error in validation
 type ErrorHandler func(w http.ResponseWriter, message string, statusCode int)
 
+// ErrorHandlerWithOpts is called when there is an error in validation, if found in Options.
+// Passes error-handling specific opts over and above the standard error handler.
+type ErrorHandlerWithOpts func(w http.ResponseWriter, message string, statusCode int, opts ErrorHandlerOpts)
+
+// ErrorHandlerArgs is used with ErrorHandlerWithOpts()
+type ErrorHandlerOpts struct {
+	*http.Request
+	*routers.Route
+	context.Context
```

Comment:
Adding a field here `error` to capture the original error would be helpful (cf: https://github.com/oapi-codegen/nethttp-middleware/issues/11#issuecomment-2548229196)